lib/plugins/encrypted_parameters/kms_key.rb

```diff
@@ -11,12 +13,27 @@ def initialize(arn)
         end
 
         def self.create
-          resp = Aws::KMS::Client.new.create_key
+          standard_tags=KmsKey.stack_tags
+          resp = Aws::KMS::Client.new.create_key({
+            tags: standard_tags, # An array of tags.
+          })
           arn = resp.key_metadata.arn
-
           new(arn)
         end
 
+        def update
+          standard_tags=KmsKey.stack_tags
```

Comment:
can be self.class.stack_tags

https://stackoverflow.com/a/12857426 has a good example on one reason why this is a convention.